app/components/admin/edition_images/uploaded_images_component.rb

```diff
@@ -39,6 +39,14 @@ def update_image_display_option_button_text
     "#{new_image_display_option == 'no_image' ? 'Remove lead' : 'Use default'} image"
   end
 
+  def lead_image_guidance
+    if @edition.type == "CaseStudy"
+      tag.p("Using a lead image is optional and can be shown or hidden. The first image you upload as the lead image.", class: "govuk-body") + tag.p("The lead image appears at the top of the document. It cannot be used in the document body.", class: "govuk-body")
```

Comment:
Should this be "is" instead of "as"?

```suggestion
      tag.p("Using a lead image is optional and can be shown or hidden. The first image you upload is the lead image.", class: "govuk-body") + tag.p("The lead image appears at the top of the document. It cannot be used in the document body.", class: "govuk-body")
```

Reply:
Good catch!